### PR TITLE
Added scripts for on-the-fly log file compression and reading.

### PR DESCRIPTION
**Pull request by dmitrijus**
Added scripts for on-the-fly log file compression (ztee.py) and reading (ztail.py).

DQMMonitoringService now outputs per-ls stats, improved event rate calculation.

Refactored esMonitoring.py to pass environment variable (including GZIP_LOG).
And added on-fly-compression option.

I had to remove -Werror=unused-variable, otherwise build fails for this package:
there is an unused variable in boost include.

### CODE REVIEW

**Comment by deguio** [comment]
I know this one was tested on the DQM playback at P5.
was it submitted for 75 as well?

**Comment by dmitrijus** [comment]
I will make port to 75 after we finish migration at P5.

**Comment by degano** [comment]
@cmsbuild please test

**Comment by deguio** [comment]
+1
please submit to 75x as well. the automatic fwd port has been disabled

File: DQMServices/StreamerIO/plugins/BuildFile.xml [hunk 1]
```diff
@@ -8,4 +8,5 @@
 
 <library   file="*.cc" name="DQMServicesStreamerIOPlugins">
   <flags   EDM_PLUGIN="1"/>
+  <flags   REM_CXXFLAGS="-Werror=unused-variable"/>
```
Comment (davidlange6) [inline DQMServices/StreamerIO/plugins/BuildFile.xml]: @dmitrijus - why is this needed?

Comment (dmitrijus) [inline DQMServices/StreamerIO/plugins/BuildFile.xml]: Hi David,

because otherwise the build fails in a boost include:
/opt/offline/slc6_amd64_gcc491/external/boost/1.51.0-cms2/include/boost/asio/error.hpp:244:45: error: 'boost::asio::error::system_category' defined but not used [-Werror=unused-variable]
 static const boost::system::error_category& system_category

Dmitrijus

**Comment by davidlange6** [comment]
+1
